pipeline_tools/input_utils.py

```diff
@@ -2,25 +2,46 @@
 from pipeline_tools.http_requests import HttpRequests
 
 
-def get_sample_id(metadata, version):
+def get_sample_id(metadata, schema_version, sequencing_protocol_id=None):
     """Return the sample id from the given metadata
 
     Args:
         metadata (dict): metadata related to sample
-        version (str): version of the metadata
+        schema_version (str): version of the metadata
 
     Returns:
         String giving the sample id
 
     Raises:
         NotImplementedError: if version is unsupported
     """
-    if version.startswith('4.'):
+    if schema_version.startswith('4.'):
         return _get_sample_id_v4(metadata)
-    elif version.startswith('5.'):
+    elif schema_version.startswith('5.'):
         return _get_sample_id_v5(metadata)
+    elif schema_version.startswith('6.'):
```

Comment:
Can you clarify here about how it could handle those verions startwith `7.`? Since I feel like `_get_smart_seq_2_fastq_names_v5_or_higher()` will possibly return `v7.0.0`?